csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/Configuration/Authenticator.cs

```diff
@@ -64,6 +66,18 @@ public Authenticator(Uri serviceUrl, string clientId, string clientSecret)
             this.serviceUrl = serviceUrl;
             this.clientId = clientId;
             this.clientSecret = clientSecret;
+
+            var handler = new HttpClientHandler();
```

Comment:
Thank you. We have almost the same code now in two places (SquidexClientManager and authenticator). Can we move this code to the options class instead and then inject the client?

Reply:
I change `Authenticator` to accept `SquidexOptions`. Now it should use `ClientFactory` which are on `SquidexOptions`. In handler case i think it should be separate because SquidexClient have authetication handler and Autheticator not and making this one abstract do not provide much value in my opinion.

Maybe timeout can be set in `IHttpConfigurator` but i am not sure if it should.